src/Razor/Microsoft.AspNetCore.Razor.Language/src/DefaultRazorCodeGenerationOptions.cs

```diff
@@ -43,6 +45,8 @@ internal class DefaultRazorCodeGenerationOptions : RazorCodeGenerationOptions
 
         public override bool OmitMinimizedComponentAttributeValues { get; }
 
+        public override bool SupportLocalizedComponentNames { get; }
```

Comment:
Nit: But just have it be a regular (get / set) property on the base type. This pattern isn't incredibly useful and I've been changing it in other places I've come across it.